gravitee-apim-console-webui/src/entities/user/userToken.fixture.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NewToken, Token } from './userTokens';
+
+export function fakeNewUserToken(name?: string): NewToken {
+  return <NewToken>{
+    name: name ?? 'J.R.R Token',
+  };
```

Comment:
Casting with `< > ` is an old TS syntax, still supported, but we should prefer `as`

```suggestion
  return {
    name: name ?? 'J.R.R Token',
  } as NewToken;
```